spec/requests/sitemap_controller_spec.rb

```diff
@@ -0,0 +1,76 @@
+require "rails_helper"
+
+RSpec.describe SitemapController do
+  subject { Nokogiri::XML(response.body) }
+
+  let(:nodes) do
+    {
+      "/ways-to-train" => {
+        "name" => "Ways to train",
+        "priority" => 0.8,
+        "date" => "2020-11-11",
+      },
+      "/salaries-and-benfits" => {
+        "name" => "Salaries and benefits",
+        "priority" => 0.7,
+      },
+      "/my-story-into-teaching/what-a-great-story" => {
+        "name" => "What a great story",
+        "priority" => 0.7,
+        "date" => "2020-10-10",
+      },
+    }
+  end
+
+  before { allow(Pages::Frontmatter).to receive(:list) { nodes } }
```

Comment:
It took me a second to realise what `nodes` were... would `pages` be clearer?

Reply:
Ah yeah, good call 👍🏽